test/Activity-test.js

```diff
@@ -109,14 +110,35 @@ describe('ActivityRepository', () => {
   });
 
   it('should return the average number of stairs climbed for all users on a given date', () => {
+    chai.spy.on(activityRepository, 'getAllUserInfoByDate', () => {
+      return [
+        {"userID": 13, "date": "2019/08/25", "numSteps": 9352, "minutesActive": 567, "flightsOfStairs": 143},
+        { "userID": 5, "date": "2019/08/25", "numSteps": 6425, "minutesActive": 67, "flightsOfStairs": 42},
+        { "userID": 6, "date": "2019/08/25", "numSteps": 842, "minutesActive": 43, "flightsOfStairs": 86 } 
+      ];
+    })
     expect(activityRepository.getAverages('2019/08/25', 'flightsOfStairs')).to.equal(90);
   });
 
   it('should return the average number of steps taken for all users on a given date', () => {
+    chai.spy.on(activityRepository, 'getAllUserInfoByDate', () => {
+      return [
+        {"userID": 13, "date": "2019/08/25", "numSteps": 9352, "minutesActive": 567, "flightsOfStairs": 143},
```

Comment:
This might be a case where you could declare this array of objects as a variable and then just pass that through every time to DRY it up but NICE! This is saweeeeeeet!

Reply:
You mean variable as in the describe block right? I agree!